pkg/static/login.js

```diff
@@ -164,17 +164,42 @@
     }
 
     function requisites() {
-        function disableLogin(name) {
+        function showBypass(bypass) {
+            if (bypass) {
+                // Selectively show and hide elements
+                show("#login", "#login-details", "#login-override");
+                hide("#get-out-link");
+
+                // Reparent login form to the expander
+                id("login-override-content").appendChild(id("login"));
+
+                // Change the state of the button from primary to warning
+                id("login-button").classList.add("pf-m-warning");
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20220929-151742-405dc998-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)